tests/unit_tests/test_credentials.js

```diff
@@ -367,18 +369,18 @@ if (!test) {
 //createWithLocalCreds('tl5h1ipgobrdqsj6.v1.p.beameio.net',{name:'Instance Information Services',email:null});
 
 switch (test) {
```

Comment:
The framework should support running specific tests. WTF this switch is doing here?

Reply:
Was already there from the "past". Didnt had time/will to redo all this credential tests and make them work/be nice. Should we do it now or open an issue to look into it separately ?

Reply:
Low priority issue, we'll do it when we do some related changes in the area.